### PR TITLE
check antenna violations in CI metrics

### LINKED ISSUE #1899: Check antenna violation metrics in the CI
### Description

Our current CI needs to check for antenna violations in its metric checks. We do not know how tool updates affect the number of antenna violations during GRT and DRT.

### Suggested Solution

After having the new antenna checker structure (OR issue https://github.com/The-OpenROAD-Project/OpenROAD/issues/3892), add antenna violations in the metrics checked in the CI. Post-DRT violations should be enough 

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by eder-matheus**
Fixes https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues/1899

### CODE REVIEW

**Comment by eder-matheus** [comment]
@vvbandeira I noticed that to have the check performed when running `make metadata,` I have to update the metrics with `make update_ok` for the design. Do I need to do this manually for every design, or do we have something in ORFS that would automatize it?

**Comment by eder-matheus** [comment]
I added the antenna violation metric to every design available in the public and secure-ci that runs in the secure branches. A few designs need to be included (like asap7/swerv_wrapper). They are not in the pipelines of the secure branches and the designs in tsmc65lp, to which I don't have access.

**Comment by vvbandeira** [comment]
> @vvbandeira I noticed that to have the check performed when running `make metadata,` I have to update the metrics with `make update_ok` for the design. Do I need to do this manually for every design, or do we have something in ORFS that would automatize it?

@habibayassin has implemented something in the dashboard, but it only works to update the metrics from the master branch. We will work on adapting the feature to update user's PRs.

**Comment by eder-matheus** [comment]
@maliberty @vvbandeira I've fixed the last conflicts in the PR. This should be ready for final review and merge.

File: flow/designs/ihp-sg13g2/aes/rules-base.json [hunk 1]
```diff
@@ -28,13 +28,17 @@
...
-        "value": 861364,
+        "value": 840803,
         "compare": "<="
     },
     "detailedroute__route__drc_errors": {
         "value": 0,
         "compare": "<="
     },
+    "detailedroute__antenna__violating__nets": {
+        "value": 252,
```
Comment (maliberty) [inline flow/designs/ihp-sg13g2/aes/rules-base.json:39]: we should work on this reducing this in a future PR

**Review by maliberty — APPROVED** [review]

**Pull request merged**